### PR TITLE
Assingment8

### PR DESCRIPTION
**Pull request by mfirmin1**

### CODE REVIEW

**Comment by mcstudor** [comment]
Will review this!

File: SecretSanta/src/SecretSanta.Web/webpack.config.js [hunk 1]
```diff
@@ -54,11 +55,25 @@ module.exports = (env, argv) => {
                     test: /\.tsx?$/,
                     loader: 'ts-loader',
                     exclude: /node_modules/
```
Comment (mcstudor) [inline SecretSanta/src/SecretSanta.Web/webpack.config.js]: Needs a comma at the end

File: SecretSanta/src/SecretSanta.Web/package.json [hunk 2]
```diff
@@ -8,6 +8,9 @@
   },
   "dependencies": {
     "bulma": "0.8.0",
+    "vue": "2.6.11",
+    "vue-class-component": "7.2.3",
+    "vue-property-decorator": "8.4.0",
```
Comment (mcstudor) [inline SecretSanta/src/SecretSanta.Web/package.json]: Missing 2 dev dependencies

File: SecretSanta/src/SecretSanta.Web/scripts/components/Gift/giftDetailComponent.vue [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    @Component
+    export default class GiftDetailsComponent extends Vue {
+        @Prop()
+        gift: Gift;
+        constructor() {
+            super();
+        }
+        mounted() {
+            let tempGift = { ...this.gift };
+            this.gift = <Gift>tempGift;
```
Comment (breaman) [inline SecretSanta/src/SecretSanta.Web/scripts/components/Gift/giftDetailComponent.vue]: this should have been a clonedGift. You don't want to overwrite the incoming property value with a new object.

**Pull request closed without merging**